__tests__/src/services/__serviceMocks__/sparrowData.json

```diff
@@ -15,19 +15,22 @@
       "gatewayUID": "dev:5678",
       "name": "My Mocked Sensor 1",
       "voltage": 2.7,
-      "lastActivity": "2021-11-30T20:30:19Z",
+      "lastActivity": "2021-11-30T14:31:24Z",
       "location": "Kitchen",
       "humidity": 27.2,
       "pressure": 101.152,
-      "temperature": 22.6
+      "temperature": 22.6,
+      "total": 281,
+      "count": 1
```

Comment:
Very cool that the hardest unit tests for this PR kind of wrote themselves since you had already set up this mock structure 👍👍

Reply:
It’s cool enough that I felt the need to leave a second comment to award a trophy emoji, which is the highest honor I can bestow in a GitHub pull request review.

🏆

Reply:
😊 😄